alpine/alpine-unfixed/types.go

```diff
@@ -0,0 +1,21 @@
+package alpineunfix
+
+type AlpineUnfix struct {
+	ID       string     `json:"id,omitempty"`
+	State    []State    `json:"state"`
+	CPEMatch []CPEMatch `json:"cpeMatch"`
+}
+
+type State struct {
+	Fixed          bool   `json:"fixed"`
+	Repo           string `json:"repo,omitempty"`
+	PackageName    string `json:"packageName,omitempty"`
+	PackageVersion string `json:"packageVersion,omitempty"`
+}
+
+type CPEMatch struct {
+	MinVersion    string `json:"minimumVersion"`
+	MinVersionOps string `json:"minimumVersionOp"`
+	MaxVersion    string `json:"maximumVersion"`
+	MaxVersionOps string `json:"maximumVersionOp"`
+}
```

Comment:
I don't think it is necessary.

Reply:
So the version that is present in state is affected version. But we can get range from CPE. Example:
![image](https://user-images.githubusercontent.com/65420264/132979200-401f8761-df06-4e86-95f4-7f281d907ffb.png)
By looking at CPE, I can tell that beep package is vulnerable **from** 0 **to** 1.3.4 version

Reply:
As I explained earlier, it is the same as NVD.